EncounterMe/EncounterMe/Services/MailService.cs

```diff
@@ -16,9 +18,16 @@ public MailService()
       
         }
 
-        public void OnSuccessfulRegistration(object source, RegistationEventArgs args)
+        public async void OnSuccessfulRegistration(object source, RegistationEventArgs args)
         {
-            MailMessage mail = new MailMessage();
+            var sendGridClient = new SendGridClient(""); //įrašyt key
```

Comment:
Kodėl tau grid labiau patinka? :D

Reply:
Nes nereikia viešai rodyt Pašto ir slaptažodžio 😄 
Šiaip iš pradžių maniau, kad eis kelt su tuo key, tai tada išvis būtų buvęs pliusas, nes nerodai tos info viešai, bet jei tą key paviešini tai užblokuoja account'ą 😄 
Tai dabar beliko pliusas tik tas, kad nereikia nurodyt žinutės turinio ir kurt viso laiško čia, nes ten kur nurodai template viskas yra saugoma ir tikriausiai gražesnis dizainas dar :D 
Dar kai tik sukūriau bendrą gmail, neleido tų settings pakeist, kur buvai link įmetus :D bet dabar pertikrinau tai jau eina :D tai spėju galima palikt ir aną jau :D

Reply:
Tai all good, o patikrinai šitas laiškus siunčia? 😄😄

Reply:
Aha, siunčia 😀
![image](https://user-images.githubusercontent.com/76535436/141801005-4d97a43e-2be0-44da-8a0f-0c1554489f46.png)

Reply:
Nuostabiai!!